src/forms/renderers/shared.ts

```diff
@@ -0,0 +1,66 @@
+import { createDefaultValue } from '@jsonforms/core';
+import { forIn } from 'lodash';
+
+export const discriminator = 'discriminator';
+
+export const getDiscriminator = (schema: any) => {
+    return schema[discriminator] ? schema[discriminator].propertyName : null;
+};
+
+export const getDiscriminatorDefaultValue = (
```

Comment:
Renamed this now that it is "shared" so folks know to only use this with a discriminator.